### PR TITLE
New Sticker Pack, all animated

### PR DESCRIPTION
**Pull request by teynav**

### CODE REVIEW

File: stickers.yml [hunk 1]
```diff
@@ -22425,3 +22425,85 @@ a92ed43943b90b6bed0f014a3d0a987d:
...
+  key: f67bd792a3a1ac00be64b6b6d21274c1e50ffbd66a52504345c1139cf8ad8a78
+  source: Telegram
+  tags:
+    - halloween
+    - duck
+    - scary
+  nsfw: false
+  original: false
+  animated: true
+
```
Comment (romainricard) [inline stickers.yml:22460]: Remove this line

File: stickers.yml [hunk 2]
```diff
@@ -22425,3 +22425,85 @@ a92ed43943b90b6bed0f014a3d0a987d:
...
+  source: Telegram
+  tags:
+    - animated
+    - joker
+    - batman
+    - DC
+  nsfw: false
+  original: false
+  animated: true
+  a92ed43943b90b6bed0f014a3d0a987d:
```
Comment (romainricard) [inline stickers.yml:22439]: Can you fix the indentation for this pack and all the following?

**Review by romainricard — CHANGES_REQUESTED** [review]

**Comment by romainricard** [comment]
Thanks for all these submissions! Please fix the error above and we'll be good to go! :+1:

**Review by teynav — COMMENTED** [review]
Sorry, it's my first time using github

**Comment by teynav** [comment]
I did both of the thing you requested, what am I doing wrong here?

**Comment by romainricard** [comment]
Thanks, the format is now correct!
There are some issues about some packs tho:

# Missing frames
Some stickers (in packs [8335949df80c37eb06c7c7872a75af68](https://signalstickers.com/pack/8335949df80c37eb06c7c7872a75af68?key=f67bd792a3a1ac00be64b6b6d21274c1e50ffbd66a52504345c1139cf8ad8a78), [293f5ffc19efa5ec5e777c5de95929d2](https://signalstickers.com/pack/293f5ffc19efa5ec5e777c5de95929d2?key=ea0f54e119172bddd959469f4d725c9ee37a43edfc310c49a28067de656bc800)) have missing frames, and thus stickers become invisible for some time, for example:

image

# Errors in layers

Some stickers (in [8335949df80c37eb06c7c7872a75af68](https://signalstickers.com/pack/8335949df80c37eb06c7c7872a75af68?key=f67bd792a3a1ac00be64b6b6d21274c1e50ffbd66a52504345c1139cf8ad8a78), [4d14ed77727bb3a53f379064c12a3ec5](https://signalstickers.com/pack/4d14ed77727bb3a53f379064c12a3ec5?key=a69794346f71ecfc533a4c86efda25167e433547a48f40112e091b48874e0a60), [82bbb0e722f3cfbfa73c981567c8f52b](https://signalstickers.com/pack/82bbb0e722f3cfbfa73c981567c8f52b?key=ff01047867757bebef2d6b3eea0cdca5da294013dc38433bf6d4e65064e11d24), [293f5ffc19efa5ec5e777c5de95929d2](https://signalstickers.com/pack/293f5ffc19efa5ec5e777c5de95929d2?key=ea0f54e119172bddd959469f4d725c9ee37a43edfc310c49a28067de656bc800)) 
have issues with layers transparency. Probably a problem of transparency.

image
image
image
image


I encourage you to take a little bit more time on your script to address these issues :smiley: 

These packs are good, and have been merged in https://github.com/signalstickers/signalstickers/commit/3f3fed51c5a1639ecee6bde5c7f0193cb3ddeddf.
https://signalstickers.com/pack/0876b357a5a8b06a192506aee2f2e09e?key=e6fa8c3db15d29c3a42e78e2f246bfedb39c8bf0034b067fd6bf5c2289f3e6b1
https://signalstickers.com/pack/249818aaf61019027e67397e6af41c4f?key=1f42ecfc4162e5b90a176911967ce6aa30e00d996b6babe72b2172122df25e02

**Comment by teynav** [comment]
Okay , this was created with older script, i will rerun it with newer one!!! Older had issues like these, although i tried to delete older packs some might have creeped in , i am converting other sticker packs rn, will convert this too!!

**Pull request closed without merging**